distribution/sdk/sample-cometd/bower.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ddf-cometd-browser-client",
+  "description": "Example of how to interact with ddf over cometd",
+  "main": "index.js",
+  "authors": [
+    "Michael O'Connor <michael.oconnor@connexta.com>"
```

Comment:
Leave off the authors tag here. We've been slowly removing them all as we go along.